integration_test/support/code_generator_case.exs

```diff
@@ -0,0 +1,84 @@
+Code.require_file("./mix_task_runner.exs", __DIR__)
+
+defmodule Phoenix.Integration.CodeGeneratorCase do
+  use ExUnit.CaseTemplate
+
+  alias Phoenix.Integration.MixTaskRunner
+
+  using do
+    quote do
+      setup_all do
+        # Get Mix output sent to the current
+        # process to avoid polluting tests.
+        Mix.shell(Mix.Shell.Process)
+      end
+
+      import unquote(__MODULE__)
+    end
+  end
```

Comment:
```suggestion
  using do
    quote do
      import unquote(__MODULE__)
    end
  end
  
  setup_all do
    # Get Mix output sent to the current
    # process to avoid polluting tests.
    Mix.shell(Mix.Shell.Process)
  end
```

Reply:
I think the current implementation doesn't need me to set the `Mix.shell` since we're running separate beam instances. If we decide to go that route, this can be removed.